telemetry/packages/public-app/components/guage.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from 'react';
+import { useQuery } from 'react-query';
+import { Chart } from 'react-google-charts';
+import { Card } from '@tremor/react';
+
+function getRandomNumber() {
+  return Math.random() * 100;
+}
+
+export function getData() {
+  return [
+    ['Label', 'Value'],
+    ['Pressure', getRandomNumber()],
+    ['Acceleration', getRandomNumber()],
+    ['Thermometer', getRandomNumber()],
+  ];
+}
+
+export function Gauge() {
+  const [data1, setData] = useState(getData);
+  const [widthX, setWidth] = useState<number | undefined>(
+    window.innerWidth < 770 ? 300 : 400,
+  );
+
+  const { data, error } = useQuery<any>(
+    'Accelerometer',
+    async () =>
+      await fetch(
+        `${process.env.NEXT_PUBLIC_TELEMETRY_SERVER}/pods/pod_1/public-data/'acceleration'?start=0`,
```

Comment:
We should really be validating the environment variables using e.g. [T3 env](https://env.t3.gg/docs/introduction). We can look at this at the next meeting :)

Reply:
I'm thinking about moving this to a separate config package which will do this globally for all packages, so we can leave this for now.